app/MyApp/index.html

```diff
@@ -227,8 +227,8 @@
     <script src='app/views/CourseStepsView.js'></script>
     <script src='app/views/BecomeMemberForm.js'></script>
     <script src='app/views/CoursesStudentsProgress.js'></script>
-    <script src='app/views/CoursesStatistics.js'></script>
-    <script src='app/views/CourseStepStatistics.js'></script>
+    <script src='app/views/CoursesStatistics.js'></script> <!-- app/views.js -->
```

Comment:
Good @Pr0chin ;)
but this is only half of the necessary changes
`app/views/CoursesStatistics.js` needs to be surrounded by double quotes `"`